src/Api/ResourceClassResolverInterface.php

```diff
@@ -25,12 +25,20 @@ interface ResourceClassResolverInterface
     /**
      * Guesses the associated resource.
      *
+     * @param mixed  $value         Object you're playing with
+     * @param string $resourceClass Resource class it is supposed to be (could be parent class for instance)
+     * @param bool   $strict        value must be type of resource class given or it will return type
+     *
```

Comment:
For next reviewers I'm to keep these comment for future developers because it's hard to guess.

Reply:
The comments need to be rewritten as they don't make much sense now.